src/EnrollOnPremController.js

```diff
@@ -89,7 +89,7 @@ function (Okta, FormType, FormController, Footer, TextBox) {
             type: 'text',
             placeholder: Okta.loc('enroll.onprem.username.placeholder', 'login', [vendorName]),
```

Comment:
placeholder doesn't have a problem?

Reply:
I've been testing this and it seems ok. Looks like Handlebars escapes the placeholder param passed (in courage).

Reply:
ah right. `placeholder` is used in the template... but tooltip is not, it is actually passed to qtip.
Makes sense then why it is not needed here.